unlock-protocol.com/blog/hello-optimistic-unlocking.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Hello Optimistic Unlocking
+subTitle: Creating a smoother, faster and a more secure online purchase experience
+authorName: Sascha Mombartz
+publishDate: April 12, 2019
+description: Introducing Optimistic Unlocking, which overcomes the slow confirmation times and provides a better user experience for blockchain transactions.
+image: /static/images/blog/hellooptimisticunlocking/hero.jpg
+---
+![Optimistic Unlocking in action.](/static/images/blog/hellooptimisticunlocking/hero.jpg)
+
+At Unlock we’re all about making transactions on the web seamless and effortless. Blockchain transactions have lots of incredible features, from low cost to permissionless, but the strong suit of blockchain technology – its safety features around immutable data also means every transaction needs to be confirmed by miners. This can take time. Generally Ethereum transactions are considered final after they’ve been confirmed by 12 miners, which can take a few minutes and longer depending on how busy the network is. This presents 2 challenges: not only is this slow, but it is very hard to predict accurately.
```

Comment:
```suggestion
At Unlock we’re all about making transactions on the web seamless and effortless. Blockchain transactions have lots of incredible features, from low cost to permissionless, but the strong suit of blockchain technology – its safety features around immutable data – also means every transaction needs to be confirmed by miners. This can take time. Generally Ethereum transactions are considered final after they’ve been confirmed by 12 miners, which can take a few minutes and longer depending on how busy the network is. This presents 2 challenges: not only is this slow, but it is very hard to predict accurately.
```